apps/desktop/tests/utils/secretLikeContent.test.ts

```
@@ -0,0 +1,66 @@
+    containsSecretLikeContent,
+    redactAllStringValues,
+    redactSecretLikeContent,
+    redactSecretLikeStringValues,
+} from '@/utils/secretLikeContent';
+
+describe('secretLikeContent', () => {
+    it('detects common secret-like substrings', () => {
+        expect(containsSecretLikeContent('OPENAI_API_KEY=sk-test-secret-value')).toBe(true);
+        expect(containsSecretLikeContent('ghp_1234567890abcdefghijklmnopqrstuvwxyzAB')).toBe(
+            true
+        );
```

ESLint

[error] Replace `⏎············true⏎········` with `true`

(prettier/prettier)